### PR TITLE
Update Chinese translations

### PR DESCRIPTION
**Pull request by TWRoach**
Update Chinese

### CODE REVIEW

**Review by TheMagnetar — APPROVED** [review]

**Comment by TheMagnetar** [comment]
Solve please the conflicts so that we can merge it

**Comment by TWRoach** [comment]
Sorry, I am a newbie. What should I do?

**Comment by TWRoach** [comment]
> Solve please the conflicts so that we can merge it

Sorry, I am a newbie. What should I do?

**Comment by TWRoach** [comment]
> Solve please the conflicts so that we can merge it

yup i'm Stupid but I think I have solved it. Right?

**Comment by TheMagnetar** [comment]
Not really, there are duplicated translations, specially in German.

File: addons/sys_rack/stringtable.xml [hunk 1]
```diff
@@ -98,6 +100,9 @@
         </Key>
         <Key ID="STR_ACRE_sys_rack_Racks">
             <English>Racks</English>
+            <German>Funkgeräteschrank</German>
```
Comment (TheMagnetar) [inline addons/sys_rack/stringtable.xml]: Duplicated translation

**Review by TheMagnetar — CHANGES_REQUESTED** [review]
For example, these are duplicated (more on that file). If you have trouble with GIT conflicts, i would suggest closing this PR and starting anew directly from the ACRE2 master branch :)

**Comment by TheMagnetar** [comment]
You are still adding duplicated entries. In sys_rack module there is
`<German>Funkgeräteschrank</German>` and `<German>Anlage</German>` for key `<Key ID="STR_ACRE_sys_rack_Racks">`

**Comment by jonpas** [comment]
@TWRoach please take another look at this PR. Looks like you were editing an older version. I tried to fix the conflicts.

**Review by TheMagnetar — APPROVED** [review]

**Pull request merged**